source/appModules/soffice.py

```diff
@@ -208,11 +209,24 @@ def announceSelectionChange(self):
 
 	def _get_cellCoordsText(self):
 		if self.hasSelection and controlTypes.State.FOCUSED in self.states:
-			selected, count = self.table.IAccessibleTable2Object.selectedCells
-			firstAccessible = selected[0].QueryInterface(IA2.IAccessible2)
+			count = self.table.IAccessibleTable2Object.nSelectedCells
+			selection = self.table.IAccessibleObject.accSelection
+			enumObj = selection.QueryInterface(oleacc.IEnumVARIANT)
+			firstChild, _retrievedCount = enumObj.Next(1)
+			# skip over all except the last element
+			enumObj.Skip(count - 2)
+			lastChild, _retrieveCount = enumObj.Next(1)

```

Comment:
```suggestion
			(lastChild, _retrievedCount): Tuple[Union[int, IDispatch], int]  = enumObj.Next(1)
```

Reply:
Same here as for the previous comment: I get an exception when applying this.

Reply:
```suggestion
			lastChild: Union[int, comtypes.client.dynamic._Dispatch]
			lastChild, _retrieveCount = enumObj.Next(1)
```